scripts/i18n.config.js

```diff
@@ -5,5 +5,7 @@ module.exports = {
         '@box/box-ai-content-answers',
         '@box/item-icon',
         '@box/metadata-editor',
+        '@box/metadata-filter',
+        '@box/metadata-view',
```

Comment:
@coderabbitai what is the purpose of this list, and when would I need to add to it?